HHS/Head-Start-TTADP
# [TTAHUB-2280] Improvements to page title format

## Description of change
Create a consistent page title format, so users can quickly identify the page purpose, as well as helping admins identify the page in Google Analytics.


## How to test
Hover over browser tab to see full page name, or use Chrome Developer tool to inspect and find <head> tag and look for page title tag.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2280


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
